Code/Framework/AzFramework/Platform/Common/Xcb/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
```

Comment:
You might need to check the 'PAL_TRAIT_LINUX_WINDOW_MANAGER_XCB' trait before even defining the target here

Reply:
Making the XCB code be in its own static library introduced a circular dependency between `AzFramework.Xcb` and `AzFramework`. So the latest revision of this PR doesn't include this file anymore.